CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+# dbt_marketo_source v0.12.0
+[PR #38](https://github.com/fivetran/dbt_marketo_source/pull/38) includes the following updates:
+
+## Breaking Change
+- Added the `action_result` field in the `stg_marketo__activity_send_email` model to capture email action outcomes, allowing for filtering in downstream models.
+  - *Note:* If you have previously added this field via the `marketo__activity_send_email_passthrough_columns` variable, remove or alias it there to prevent duplicate column errors.
```

Comment:
```suggestion
  - *Note:* If you have previously added this field via the `marketo__activity_send_email_passthrough_columns` variable, remove or alias it there to prevent duplicate column errors. See the [README](https://github.com/fivetran/dbt_marketo_source?tab=readme-ov-file#passing-through-additional-columns) for more details on working with passthrough columns.
```